static/styles/compose.css

```diff
@@ -230,6 +249,7 @@
     display: flex;
     justify-content: space-between;
     padding-bottom: 5px;
+    height: 24px; /* to keep height consistent between PM and stream messages */
```

Comment:
Possibly this comment isn't really needed. I can put this change in its own commit with that as the commit message?

Reply:
Yeah, that's an excellent case for a small prep commit. I'd then also consider tweaking that commit to do something like 2268cb84775d6dfb650a18f6ca2ed2cf0f5f5bcf.

Reply:
I love using variables for magic numbers! Would it be helpful to make changes like that in parts of the codebase I touch, more generally?

Reply:
Oh actually this doesn't work on its own. It's related to the issue I'm discussing with Vlad in design right now.